app/javascript/Admin/Admin/Request/Organization.elm

```
@@ -40,3 +43,17 @@ requestOrganization env apiKey organizationId =
             , timeout = Nothing
             , withCredentials = False
             }
+
+requestCreateOrganization : OrganizationData -> Reader NodeEnv (Task GQLClient.Error Organization)
+requestCreateOrganization orgInputs =
+    Reader.Reader
+        (\env ->
+            GQLClient.sendMutation (graphqlUrl env) <|
```

Ok, here is a catch, this mutation might work in development mode but it will fail on staging, as you are not passing organization API key. Use `customSendMutation` instead of `sendMutation`.

---

@sudeeptarlekar Why is that it will be work in development and while it won't work in staging. We should fix our tool chain so that something that will break in staging will also break in development. Can you create another issue and do a fix there.

---

but the mutation I'm sending here is to create the organization right after the Signup. The user will not be any part of Organization.

---

Problem is in `load_organization.rb` concern where we are loading first organization by default if in development mode and this is suppressing the error on local.